lib/mailserver.js

```
@@ -169,14 +173,17 @@ mailServer.createElasticsearchClient = function (host, port) {
  * Start the mailServer
  */
 
-mailServer.listen = function(port){
+mailServer.listen = function(port, elasticSearchInfo){
     
     // Start the server & Disable DNS checking
     smtp = simplesmtp.createServer({
         disableDNSValidation: true
     });
 
-    mailserver.elasticsearchClient = mailServer.createElasticsearchClient(mailServer.elasticsearchConfig.host, 'trace');
+    if(elasticSearchInfo) {
```

should this fail loudly if there is no elasticsearch info?